ui/app/mainui/AppMain.qml

```diff
@@ -828,10 +851,8 @@ Item {
         active: appMain.rootStore.mainModuleInst.sectionsLoaded
         sourceComponent: StatusStickersPopup {
             store: appMain.rootChatStore
-            walletAssetsStore: appMain.walletAssetsStore
-            sendModalPopup: sendModal
-
             isWalletEnabled: appMain.profileStore.isWalletEnabled
+            onBuyClicked: Global.buyStickerPackRequested(packId, price)
```

Comment:
Do we need the `Global` call here? we could call the sendModalHandler directly

Reply:
thats a miss!! will push a fix now..